src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Arrays.stream(Color.values()).findAny().get().name();
```

Comment:
The use of `findAny()` does not guarantee a random selection from the `Color` enum. Consider using a `Random` object to select a random index from the `Color.values()` array to ensure randomness. For example, `Color.values()[new Random().nextInt(Color.values().length)].name()`.